### PR TITLE
automated email template

### PR DESCRIPTION
**Pull request by michaelyaacoub**
I finished the automated email! Tried to test the code, and it seems to be working. https://github.com/tdowds/rememberthee-website/issues/18

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -37,4 +37,10 @@ def sign_up():
 	""".format(**data)
 	mail.send(message)
 
-	return jsonify(success=True)
+	subject= "Thank you for siging up for Remember Thee"
+	message = Message(subject, sender='trevor@rememberthee.com', recipients=[data['email']])
+	message.html = render_template('welcome_email.html', firstName=data['firstName'])
+	mail.send(message)
+	print('success!!')
```
Comment (tdowds) [inline app.py:44]: Let's remove this `print` statement

File: templates/index.html [hunk 2]
```diff
@@ -233,6 +233,7 @@ <h3><u>Contact Us</u></h3>
           submit_button.disabled = false;
           submit_button.innerHTML = "Submit";
           submit_button.style.opacity = 1;
+          submit_button.style.padding = 10;
```
Comment (tdowds) [inline templates/index.html:236]: Why is this added in? Not seeing any difference between when I leave it in and take it out

File: templates/welcome_email.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+
+<body>
+    <h3>Hi {{firstName}},</h3>
```
Comment (tdowds) [inline templates/welcome_email.html:9]: Let's not wrap this in a header element; `<p>` should probably be fine, but will want a space between it and the paragraph starting below it.

File: templates/welcome_email.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    <p>Thanks for filling out the form on <a href="https://www.rememberthee.com" target="_blank">rememberthee.com</a>.
+        Remember Thee is available for download on iOS only. <br>You will receive an email from Apple's app testing
```
Comment (tdowds) [inline templates/welcome_email.html:11]: Don't think we need the `<br>`, why did you include it?

File: templates/welcome_email.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    <ul>
+        <li>This version of Remember Thee does not contain all the features you saw on the website.
+            <br>They will be added in the coming days and weeks. If there was a feature you were particularly excited
+            about, please let me know.
+        </li><br>
+        <li>In general, if there are things you wish the app could do, or if it's not really working for you, please let
+            me know.
+        </li><br>
+        <li>If the app crashes, something looks visibly broken or doesn't work or is difficult to use, or you're having
+            trouble downloading the app. <br>
```
Comment (tdowds) [inline templates/welcome_email.html:25]: Please remove the `<br>` - this list item is one (long) sentence, so should be a comma after the word "app", followed by "you can reach out...."

File: templates/welcome_email.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    <h3>Hi {{firstName}},</h3>
+    <p>Thanks for filling out the form on <a href="https://www.rememberthee.com" target="_blank">rememberthee.com</a>.
+        Remember Thee is available for download on iOS only. <br>You will receive an email from Apple's app testing
+        product, <a href="https://testflight.apple.com" target="_blank">TestFlight</a>, with instructions on how to
+        download Remember Thee shortly.</p>
+
+    <p>A few things to keep in mind while downloading and using Remember Thee:</p>
+    <ul>
+        <li>This version of Remember Thee does not contain all the features you saw on the website.
+            <br>They will be added in the coming days and weeks. If there was a feature you were particularly excited
```
Comment (tdowds) [inline templates/welcome_email.html:18]: Don't think we need `<br>` here either. Let it the sentences wrap on their own.

File: templates/welcome_email.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+        </li><br>
+        <li>In general, if there are things you wish the app could do, or if it's not really working for you, please let
+            me know.
+        </li><br>
+        <li>If the app crashes, something looks visibly broken or doesn't work or is difficult to use, or you're having
+            trouble downloading the app. <br>
+            You can reach out to me either at this email or at help@rememberthee.com, and I'll get back to you as soon
+            as I can to help fix the problem.
+        </li><br>
+        <li>If you're enjoying the app, please feel free to share the link to the website with others. <br>
```
Comment (tdowds) [inline templates/welcome_email.html:29]: No `<br>` needed here either, let the sentences wrap on their own

File: templates/welcome_email.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+            Anyone that signs up on the website will pretty much be given access to the beta version of the app.
+        </li>
+        <br>
+    </ul>
+    <p>Thanks, and let me know if you have any further questions, comments or problems. I hope Remember Thee will be
+        blessing to you either now, or in the future.</p>
+
+    <h5>Trevor Dowds, <br>Founder and Creator</h5>
+    <br>
+    <img src="/static/images/Logo.png">
```
Comment (tdowds) [inline templates/welcome_email.html:39]: When I tested this out, this failed (see screenshot below). You probably need to specify the full path in the `` element - `src="https://www.rememberthee.com/static/images/Logo.png"`

Screen Shot 2021-11-18 at 9 47 23 AM

**Comment by tdowds** [comment]
Great work so far @michaelyaacoub - a few more comments and we're almost there on this one. Thanks!

File: templates/welcome_email.html (hunk 4 above)
Comment (michaelyaacoub) [reply templates/welcome_email.html:11]: The <p> was on one line and it was too long. Do you want to earise?

File: app.py [hunk 9]
```diff
@@ -37,4 +37,9 @@ def sign_up():
 	""".format(**data)
 	mail.send(message)
 
-	return jsonify(success=True)
+	subject= "Thank you for siging up for Remember Thee"
```
Comment (tdowds) [inline app.py:40]: "siging" should be "signing"

Comment (michaelyaacoub) [reply app.py:40]: 🤦‍♂️sorry my bad 😅

File: templates/welcome_email.html (hunk 4 above)
Comment (tdowds) [reply templates/welcome_email.html:12]: Yes, get rid of the `<br>`, you see how the text wraps around on its own once I get rid of it from the code?
Screen Shot 2021-11-18 at 10 18 30 AM

File: templates/welcome_email.html [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+        <li>This version of Remember Thee does not contain all the features you saw on the website.
+            They will be added in the coming days and weeks. If there was a feature you were particularly excited
+            about, please let me know.
+        </li>
+        <li>In general, if there are things you wish the app could do, or if it's not really working for you, please let
+            me know.
+        </li>
+        <li>If the app crashes, something looks visibly broken or doesn't work or is difficult to use, or you're having
+            trouble downloading the app,
+            You can reach out to me either at this email or at help@rememberthee.com, and I'll get back to you as soon
```
Comment (tdowds) [inline templates/welcome_email.html:27]: lower case "y" in "You can reach out...". Also move this up to after "app," in the line above, please

File: templates/welcome_email.html [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+            as I can to help fix the problem.
+        </li>
+        <li>If you're enjoying the app, please feel free to share the link to the website with others.
+            Anyone that signs up on the website will pretty much be given access to the beta version of the app.
+        </li>
+        <br>
```
Comment (tdowds) [inline templates/welcome_email.html:33]: You should put this `<br>` below the the `</ul>`

File: templates/welcome_email.html [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+    </ul>
+    <p>Thanks, and let me know if you have any further questions, comments or problems. I hope Remember Thee will be
+        blessing to you either now, or in the future.</p>
+
+    <h5>Trevor Dowds, <br>Founder and Creator</h5>
+    <br>
+    <img src="https://www.rememberthee.com/static/images/Logo.png">
```
Comment (tdowds) [inline templates/welcome_email.html:40]: Works now but the image is too large, can you scale down to 175x36? and remove the `<br>` above it?

**Comment by tdowds** [comment]
This looks good to me @michaelyaacoub - I pushed up a commit to do a little bit of code clean up, but otherwise really great work. Thank you!
